src/lib/components/HttpErrors/HttpErrors.js

```diff
@@ -0,0 +1,64 @@
+import Overridable from 'react-overridable';
+import React from 'react';
+import HttpError from './HttpError';
+
+function UnauthorizedComponent(props) {
+  return (
+    <Overridable id="Unauthorized.layout" {...props}>
+      <HttpError
+        title="Unauthorized"
+        message="You are not authorized to access this resource."
+        icon="ban"
+      />
+    </Overridable>
+  );
+}
+
+function ForbiddenComponent(props) {
+  return (
```

Comment:
Do you believe that giving the ability to override the values with props would make sense? Something like 

```
const {
        title="Access Forbidden"
        message="You do not have the sufficient privileges to access this resource."
        icon="ban"
} = this.props
```
and then 
```
      <HttpError
        title={title}
        message={message}
        icon={icon}
      />
```
So instead of redefining a component we could do
`<Forbidden title="My special forbidden title for this case">`

Reply:
Yes it makes sense, good suggestion!